contracts/Orders.sol

```diff
@@ -1,56 +1,31 @@
 pragma solidity 0.4.18;
 
-
 import "./Utils2.sol";
+import "./Withdrawable.sol";
 
-
-contract Orders is Utils2 {
+contract Orders is Withdrawable, Utils2 {
 
     struct Order {
         address maker;
         uint32 prevId;
         uint32 nextId;
-        uint8 data;
         uint128 srcAmount;
         uint128 dstAmount;
     }
 
     mapping (uint32 => Order) public orders;
 
-    uint8 constant public ETH_TO_TOKEN = 1;
-    uint8 constant public TOKEN_TO_ETH = 2;
-
-    uint32 constant public TAIL_ID = 1;
-    uint32 constant public BUY_HEAD_ID = 2;
-    uint32 constant public SELL_HEAD_ID = 3;
+    uint32 constant public TAIL_ID = 0;
```

Comment:
don't like using 0. unless u r sure its best practice.
since all is init by default to 0. if we skip some init we think it points on tail when we don't mean it!!!

Reply:
Done.